admin.php

```diff
@@ -8,7 +8,7 @@
 
 if(!defined('DOKU_INC')) die();
 
-require_once (DOKU_INC.'inc/HTTPClient.php');
+require_once (DOKU_INC.'inc/HTTP/HTTPClient.php');
```

Comment:
Could you try to just remove this include? It should be autoloaded.
(See also https://github.com/samuelet/indexmenu/issues/214#issuecomment-637777181 )

Reply:
My understanding is that at least “ require_once (DOKU_INC.'inc/HTTPClient.php');” can be removed. That is better then updating the path, because your change is not backward compatible.

Maybe that other line can be removed as well, I’m not yet sure that the unzip is still working now.
Did you test?

(Btw DokuWiki includes today also a unzip class, so this zip library can probably be replaced)